quantuminspire/cli/command_list.py

```diff
@@ -335,6 +335,17 @@ def login(
     typer.echo(f"Using member ID {settings.auths[host].team_member_id}")
 
 
+@app.command("set-default-host")
+def set_default_host(
+    host: str = typer.Argument(help="The URL of the platform to which to connect"),
+) -> None:
+    """Set default_host for interacting with Quantum Inspire."""
+    settings = Settings()
+    settings.default_host = Url(host)
```

Comment:
I think we should have client IDs mapped to host urls

Reply:
Isnt this already done in the `DEFAULT_CONFIG` json (see configuration.py)?

Reply:
ah yes, you are right